src/components/Missionlist.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-unused-vars */
+import { React } from 'react';
+import { PropTypes } from 'prop-types';
+
+const MissionsList = ({ id, name, description }) => (
+  <>
+    <table className="table">
```

Comment:
I think you don't need to create a new table for each mission, but only a new row for the table:
`<tr key={id}>
          <td>{name}</td>
          <td>{description}</td>
        </tr>`